lib/Signaling/Listener.php

```diff
@@ -51,7 +53,9 @@ public static function register(IEventDispatcher $dispatcher): void {
 
 	protected static function isUsingInternalSignaling(): bool {
 		$config = Server::get(Config::class);
-		return $config->getSignalingMode() === Config::SIGNALING_INTERNAL;
+		$isSignalingInternal = $config->getSignalingMode() === Config::SIGNALING_INTERNAL;
+		$isSignalingDev = $config->isSignalingDev();
+		return $isSignalingInternal && !$isSignalingDev;
```

Comment:
This breaks testing external signaling server in dev mode. As mentioned please just ignore the fact for now.